src/components/Hotels/Hotels.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint linebreak-style: ["error", "windows"] */
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { FetchedHotels } from '../../redux/hotels/hotels';
+import Hotel from '../Hotel/Hotel';
+import './Hotels.css';
+
+const Hotels = () => {
+  const dispatch = useDispatch();
+  const hotels = useSelector((state) => state.hotels.data);
+  console.log(hotels);
```

Comment:
- [ ] Kindly remove this old piece of debugging console.log statement, in order to improve the repo's cleanliness. 👍